test/end-to-end/offer.js

```diff
@@ -233,47 +343,60 @@ describe("Offer endpoint tests", () => {
 
         describe("Using already created offer(s)", () => {
             describe("Only current offers are returned", () => {
-                const test_offer = {
-                    title: "Test Offer",
-                    publishDate: "2019-11-22T00:00:00.000Z",
-                    publishEndDate: "2019-11-28T00:00:00.000Z",
-                    description: "For Testing Purposes",
-                    contacts: { email: "geral@niaefeup.pt", phone: "229417766" },
-                    jobType: "SUMMER INTERNSHIP",
-                    fields: ["DEVOPS", "MACHINE LEARNING", "OTHER"],
-                    technologies: ["React", "CSS"],
-                    owner: "aaa712371273",
-                    location: "Testing Street, Test City, 123",
-                };
-
-                const expired_test_offer = {
-                    title: "Expired Test Offer",
-                    publishDate: "2019-11-17",
-                    publishEndDate: "2019-11-18",
-                    description: "For Testing Purposes",
-                    contacts: { email: "geral@niaefeup.pt", phone: "229417766" },
-                    jobType: "SUMMER INTERNSHIP",
-                    fields: ["DEVOPS", "MACHINE LEARNING", "OTHER"],
-                    technologies: ["React", "CSS"],
-                    owner: "aaa712371273",
-                    location: "Testing Street, Test City, 123",
-                };
 
-                const future_test_offer = {
-                    title: "Future Test Offer",
-                    publishDate: "2019-12-12",
-                    publishEndDate: "2019-12-22",
-                    description: "For Testing Purposes",
-                    contacts: { email: "geral@niaefeup.pt", phone: "229417766" },
-                    jobType: "SUMMER INTERNSHIP",
-                    fields: ["DEVOPS", "MACHINE LEARNING", "OTHER"],
-                    technologies: ["React", "CSS"],
-                    owner: "aaa712371273",
-                    location: "Testing Street, Test City, 123",
-                };
```

Comment:
Any reason to move the declaration inside the `beforeAll` hook and have (possibly mutable) higher-scoped variables? The variables need to be in scope for all the tests below, so why not have them as `const` where they were?

Reply:
initialization problems, should be fixed now